nomic/pl_callbacks/pl_callback.py

```diff
@@ -45,8 +52,17 @@ def clear(self):
         self.embeddings = []
         self.metadata = defaultdict(list)
 
+
 class AtlasEmbeddingExplorer(Callback):
-    def __init__(self, max_points=-1, rebuild_time_delay=600, name=None, description=None, is_public=True, overwrite_on_validation=False):
+    def __init__(
+        self,
+        max_points=-1,
```

Comment:
As part of docs, can we include what -1 is referring to here?

Reply:
done